### DISCUSSION TITLE
I built a platform to answer code questions about Amplication

### DISCUSSION THREAD

**Discussion by SantiagoUriarte** [amplication/amplication#5900] [Show and tell]
Hey everyone I built a platform to help developers get answers to their code-related questions. We trained the platform on Amplication's codebase so you can ask it general code-related questions as well as domain-specific questions.

It's free to use and if you want access to it just sign up here and we will email you the credentials
https://7syy9yo4wq0.typeform.com/to/xDWkEw1i
While using the product, I'd love to hear any suggestions or feedback about how to make the product better.

If you want to see the product in action here is a short demo:
https://youtu.be/CclaGk2Zj6Q

**Comment by MichaelSolati** [comment]
> Hey everyone I built a platform to help developers get answers to their code-related questions. We trained the platform on Amplication's codebase so you can ask it general code-related questions as well as domain-specific questions.
> 
> It's free to use and if you want access to it just sign up here and we will email you the credentials
> [https://7syy9yo4wq0.typeform.com/to/xDWkEw1i](url)
> While using the product, I'd love to hear any suggestions or feedback about how to make the product better.
> 
> 
> If you want to see the product in action here is a short demo:
> https://youtu.be/CclaGk2Zj6Q

Typeform link doesn't work, do you mind resharing it.

**Reply by SantiagoUriarte** [reply]
Thanks for pointing that out I edited and it should work now
